tests/modules/core/lib/Auth/UserPassBaseTest.php

```diff
@@ -60,7 +67,8 @@ public function testAuthenticateECPMissingUsername()
      */
     public function testAuthenticateECPMissingPassword()
     {
-        $this->expectException(\SimpleSAML\Error\Error::class, 'WRONGUSERPASS');
+        $this->expectException(\SimpleSAML\Error\Error::class);
+        $this->expectExceptionMessage('WRONGUSERPASS');
 
```

Comment:
Not sure in what PHPunit-version this was allowed, but today it's split in two calls (actually for years already)